host.go

```diff
@@ -305,7 +306,44 @@ func (h *gpbftHost) ReceiveDecision(decision *gpbft.Justification) time.Time {
 		return time.Now().Add(h.manifest.ECDelay)
 	}
 
-	return ts.Timestamp().Add(h.manifest.ECDelay)
+	if !decision.Vote.Value.IsOnlyBase() {
+		// we decided on something new, use just the ECDelay
+		return ts.Timestamp().Add(h.manifest.ECDelay)
+	}
+
+	// we decided on base, calculate how much we should back off
+	// all of this should go into manifest but I think Alfonso dislikes me already :P
+	const (
+		minBackoff = 2.
+		maxBackoff = 3. * 2. * 60. // 3h with 30s ECDelay
+	)
+	// the backoff is defined in multiples of ECDelay starting at the last finalized tipset
+	// each additional base decision beyond that will incurr the maxBackoff
+	var backoffTable = []float64{2, 3, 5, 9, 17, 33, 65, 129, 257, 513} // 2^i+1 backoff, table for more flexibility
```

Comment:
Maybe formulate this similar to [capped rebroadcast backoff](https://github.com/filecoin-project/go-f3/blob/main/gpbft/options.go#L116)?

Reply:
I went with a table because it gives us maximum flexibility with the manifest system. We can switch to a formula in the future.
It is also a bit different, as we are backing off based on the same finalized tipset, not `time.Now()`. Also the formulation of each instance causing X time of backoff is more intuitive IMO.